src/poliastro/earth/sensors.py

```diff
@@ -0,0 +1,121 @@
+
+    R = Earth.R.to(u.km)
+    r_sat = R + h
+    n_max = n_center + n_fov / 2
+    n_min = n_center - n_fov / 2
+    γ_max = np.arcsin(r_sat * np.sin(n_max) / R)
+
+    if abs(γ_max) <= np.pi / 2 * u.rad:
+        γ_max = np.pi * u.rad - γ_max
+    γ_min = np.arcsin(r_sat * np.sin(n_min) / R)
+    if abs(γ_min) <= np.pi / 2 * u.rad:
+        γ_min = np.pi * u.rad - γ_min
+
+    p_max = R * np.cos(γ_max) + r_sat * np.cos(n_max)
+    p_min = R * np.cos(γ_min) + r_sat * np.cos(n_min)
+    Λ_max = np.arcsin(p_max * np.sin(n_max) / R)
+    Λ_min = np.arcsin(p_min * np.sin(n_min) / R)
+
+    return Λ_max, Λ_min
+
+
+@u.quantity_input(
+    h=u.km, n_fov=u.rad, n_center=u.rad, β=u.rad, φ_nadir=u.rad, λ_nadir=u.rad
+)
+def max_and_min_ground_range_with_specific_azimuth(
+    h, n_fov, n_center, β, φ_nadir, λ_nadir
+):
+    """
+    Calculates the difference in ground-range angles from the n_center angle and the latitude and longitude of the target
+    for a desired phase angle,:math:`{\beta}`, used to specify where the sensor is looking.
+
+    Parameters
+    ----------
+    h: ~astropy.units.Quantity
+        Altitude over surface.
+    n_fov: ~astropy.units.Quantity
+        Angle of the total area that a sensor can observe.
+    n_center: ~astropy.units.Quantity
+        Center boresight angle.
+    :math:`{\beta}`: ~astropy.units.Quantity
+        Phase angle, used to specify where the sensor is looking.
+    φ_nadir: ~astropy.units.Quantity
+        Latitude angle of nadir point.
+    λ_nadir: ~astropy.units.Quantity
+        Longitude angle of nadir point.
+
+    Returns
+    -------
+    delta_λ : ~astropy.units.Quantity
+        The difference in ground-range angles from the n_center angle.
+    φ_ctr: ~astropy.units.Quantity
+        Latitude angle of the target point.
+    λ_ctr: ~astropy.units.Quantity
+        Longitude angle of the target point.
+
+    Raises
+    ------
+    ValueError
+        This formula always gives the answer for the short way to the target ot the acute angle,:math:`{\beta}`,
+        which must be greater than 0º and less than 180º.
+
+    Notes
+    -----
+    For further information, please take a look at "Fundamentals of Astrodynamics and Applications", 4th ed (2013)"
+    by David A. vallado, pages 853-860.
+
+    """
+    if 0 * u.rad < β > np.pi * u.rad:
+        raise ValueError("β must be between 0º and 180º")
+
+    R = Earth.R.to(u.km)
+    r_sat = R + h
+
+    γ = np.arcsin(r_sat * np.sin(n_center) / R)
+    if abs(γ) <= np.pi / 2 * u.rad:
+        γ = np.pi * u.rad - γ
+
+    p = R * np.cos(γ) + r_sat * np.cos(n_center)
+    Λ = np.arcsin(p * np.sin(n_center) / R)
+    φ_ctr = np.arcsin(
+        np.cos(β) * np.cos(φ_nadir) * np.sin(Λ) + np.sin(φ_nadir) * np.cos(Λ)
+    )
+    delta_Λ = np.arcsin(np.sin(β) * np.sin(Λ) / np.cos(φ_ctr))
+    λ_ctr = λ_nadir + delta_Λ
+    Λ_max, Λ_min = max_and_min_ground_range(h, n_fov, n_center)
+    delta_λ = (Λ_max - Λ_min) / 2
+
+    return delta_λ, φ_ctr, λ_ctr
```

Comment:
I'm following Example 11-1 from Vallado, shouldn't we call these `φ_tgt` and `λ_tgt`?